internal/ingress/metric/collectors/socket.go

```diff
@@ -132,13 +135,23 @@ func NewSocketCollector(pod, namespace, class string, metricsPerHost, reportStat
 		requestTags = append(requestTags, "host")
 	}
 
+	em := make(map[string]struct{}, len(excludeMetrics))
+	for _, m := range excludeMetrics {
+		// remove potential nginx_ingress_controller prefix from the metric name
+		// TBD: how to handle fully qualified histogram metrics e.g. _buckets and _sum. Should we just remove the suffix and remove the histogram metric or ignore it?
```

Comment:
@tao12345666333 What do you recommend regarding the best way how to handle the fully qualified histogram metrics including the _buckets suffix e.g. `nginx_ingress_controller_response_duration_seconds_bucket`?

I've added documentation in the `cli-arguments.md` docs so users can actually figure out what the correct metric names are and in my opinion we should just remove this comment and ignore metrics ending with _buckets or _sum.